src/pages/IDO/styled.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+
+const handleColorType = (status?: any, theme?: any) => {
+  switch (status) {
+    case 'pending':
+      return theme.blue1
+    case 'active':
+      return theme.blue1
+    case 'succeeded':
+      return theme.green1
+    case 'defeated':
+      return theme.red1
+    case 'queued':
+      return theme.text3
+    case 'executed':
+      return theme.green1
+    case 'canceled':
+      return theme.text3
+    case 'expired':
+      return theme.text3
+    default:
+      return theme.text3
+  }
+}
+
+export const ProposalStatus = styled.span<{ status: string }>`
```

Comment:
Are we using this? Seems like boilerplate from the Vote page

Reply:
removed the IDO/styled.tsx file. It's not used right now